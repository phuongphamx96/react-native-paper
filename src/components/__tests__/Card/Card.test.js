import React from 'react';
import renderer from 'react-test-renderer';
import Card from '../../Card/Card';

describe('Card', () => {
  it('renders an outlined card', () => {
    const tree = renderer.create(<Card mode="outlined" />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
