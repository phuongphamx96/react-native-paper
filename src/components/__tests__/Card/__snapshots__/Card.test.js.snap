// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card renders an outlined card 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#ffffff",
      "borderColor": "rgba(0, 0, 0, 0.12)",
      "borderRadius": 4,
      "borderWidth": 1,
      "elevation": 0,
    }
  }
>
  <View
    accessible={true}
    focusable={false}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "flexGrow": 1,
        "flexShrink": 1,
      }
    }
  />
</View>
`;
